source/components/github.rst

```diff
@@ -0,0 +1,51 @@
+***********************
+Using GitHub at NSLS-II
+***********************
+
+Two-factor authentication
+-------------------------
+
+We strongly recommend `securing your account with two-factor authentication <https://help.github.com/en/articles/securing-your-account-with-two-factor-authentication-2fa>`_.
+
+For normal use from the campus network, the most convenient way to access
+GitHub is using SSH. See `GitHub's SSH guide <https://help.github.com/en/articles/connecting-to-github-with-ssh>`_.
+Your remotes will look like:
+
+.. code-block:: bash
+
+
+   $ git remote -v
+   danielballan   git@github.com:danielballan/bluesky (fetch)
+   danielballan   git@github.com:danielballan/bluesky (push)
+   origin         git@github.com:bluesky/bluesky (fetch)
+   origin         git@github.com:bluesky/bluesky (push)
+
+Note ``git@github.com:`` in place of where you might have
+``https://github.com/``. You can update a remote using
+
+.. code-block:: bash
+
+   $ git remote set-url <NAME> <NEW_URL>
+
+For use inside the ring, connecting via SSH does not work. (Consult ITC to ask
```

Comment:
Not sure if this is intended?
```suggestion
For users inside the ring, connecting via SSH does not work. (Consult ITC to ask
```

Reply:
Fair question. I meant "use". Maybe it could be phrase more clearly?

Reply:
No, don't put users inside the ring, it's not safe there 🤣.

Maybe we can better say "For use in the controls network ..."?